app/api/permissions/specs/entitiesPermissions.spec.ts

```diff
@@ -103,5 +176,24 @@ describe('permissions', () => {
         },
       ]);
     });
+
+    describe('publicly shared', () => {
+      it('should return a permission of type "public" and level "mixed" if the entities are published and unpublished', async () => {
+        const permissions = await entitiesPermissions.get(['shared1', 'shared4']);
+        expect(permissions).toEqual(
+          expect.arrayContaining([{ ...publicPermission, level: MixedAccess.MIXED }])
```

Comment:
Shouldnt `MixedAccess.MIXED` be a constant of `Accesslevels`

Reply:
@konzz  It should, and should not :D 
The thing is: We should have two enums, one with and one without `MIXED`; because in some contexts mixed is allowed and in some others it isn't. But there's a limitation in TS for which you cannot 'extend' enums, and having duplicated values just because we wan't those two scenarios made no sense. So the solution is to have one enum for real permissions, one enum for the contextual permission (mixed access) and a type that merges both for typing things. Sadly, you cannot access the enum value from that merge. So that's why we use both enums to get values.

Reply:
👍